src/components/MoneyRequestHeader.js

```diff
@@ -65,7 +62,8 @@ const defaultProps = {
     policy: {},
 };
 
-function MoneyRequestHeader({session, parentReport, report, parentReportAction, transaction, policy, personalDetails}) {
+function MoneyRequestHeader({session, parentReport, report, parentReportAction, transaction, policy}) {
+    const personalDetails = usePersonalDetails() || CONST.EMPTY_OBJECT;
```

Comment:
Can't we move `EMPTY_OBJECT` to the `usePersonalDetails` hook itself?

Reply:
I am not sure of it because it was already introduced in the codebase and I have just copy/ pasted it from elsewhere. Apart from this, The hook is coming from `Expensify-App/src/components/OnyxProvider.tsx` so I am not sure whether there's a straight forward way to do that.

I can ask @kacper-mikolajczak if that's possible because he did investigate around this.

Reply:
Hi @allroundexperts @hurali97, I've done that quite a while ago but it probably still holds true. 

The `usePersonalDetails` hook was a replacement for passing the `personalDetails` via props. Depending on a scenario, `personalDetails` prop sometimes had fallback in form of **defaultProps** and sometimes not.
In order to have full compat between those usages and not break anything, I decided to inline fallback to empty object depending on a use-case. 

Hope that helps 👍